README.md

```diff
@@ -127,13 +128,17 @@ ConditionPathExists=/usr/local/bin/palomad
 [Service]
 Type=simple
 Restart=always
+RestartSec=5
 WorkingDirectory=~
 ExecStartPre=
-ExecStart=/usr/local/bin/palomad start --p2p.persistent_peers da10f3078f78dabbb38b59df2c90230716f3149c@157.245.76.119:26656
+ExecStart=/usr/local/bin/palomad start --p2p.persistent_peers f64dd167410a242c993648faa6406edf74a7f4b7@157.245.76.119:26656
```

Comment:
@Vizualni Is there a way to remove the `--p2p...` part? I thought adding addrbook.json would do this.

Reply:
It should work with the address book. I haven't really played with the address book so I wouldn't know at this point :(. Initially I wanted to use addresbook, but it didn't work for me so I've decided to use the --p2p arguments instead and leave the addressbook for future Matija to figure out :)